Jamfile

```diff
@@ -766,6 +770,7 @@ lib torrent
 
 	<dht>on:<source>src/kademlia/$(KADEMLIA_SOURCES).cpp
 	<dht>on:<source>ed25519/src/$(ED25519_SOURCES).cpp
+	<dht>on:<source>src/hasher512.cpp
```

Comment:
not sure, there are a couple of details here, 1- clang complains about an empty object, 2- `hasher512` is only relevant if `DHT` is enable, but it's not exactly part of it. You decide.